### ISSUE TITLE
BUG: properties from other worlds appear on property list, causes infinite crash loops

### ISSUE THREAD

**Issue by badooga** [DarkflameUniverse/DarkflameServer#323]
### Make sure you've done the following:

- [X] I have checked that this issue has not already been reported.

- [X] I have validated that this issue is not a syntax error of either MySQL or SQLite.

- [X] I have pulled the latest version of the main branch of DarkflameServer and have confirmed that the issue exists there.

### Issue Description

When you go to a property launcher, properties from other worlds that other people made show up, in addition to the ones they made for the intended world (which I will call "ghost properties", for reasons stated below). For example, one character had two entries on the property listings for Chantey Shantey - one for their Block Yard property, and another for their Nimbus Rock property.

When I attempted to visit the ghost Nimbus Rock property from the Chantey Shantey launcher, it instead brought me to an empty version of Chantey Shantey that I could "claim" and build on. Note that this area didn't count towards the "visit 5 properties" mission I had. I then left without incident. However, I then asked the ghost property owner to actually claim Chantey Shantey. They did so, and then when I attempted to revisit the same ghost property as before, I crashed on the loading screen, and any attempt to re-enter that character sends me back to the log-in screen (similar to #156). They then traveled to Nimbus Station to make their actual Nimbus Rock private, but attempting to travel there sent them into an infinite crash loop as well.

### Reproduction steps

1. Have a friend claim a single property (A), make it public, and get it approved. Have them name it after the name of the world for ease of testing.
2. Go to a different property launcher (B) that your friend hasn't claimed yet, check the listings, and notice that property A is listed there. Attempt to visit property A and notice how it is empty and includes a console you can use to "claim" and build on it.
3. Have your friend claim property B and make it public. If you attempt to revisit property A from the same location, you'll get into an infinite crash loop. Similarly, if your friend attempts to visit property A from its actual location, they will get into an infinite crash loop too.

### Expected Behavior

1. When at a property launcher, properties from other worlds should not appear on the list of options (presumably; idk if this was intentional or not).
2. The game should run without infinite crash loops.

### Environment

Ubuntu

**Comment by metric-ambient** [comment]
I have also had this issue with properties from other worlds appearing on launch stations they shouldn't. It can cause some serious issues, like the one noted above. Would really love a fix for this.

**Comment by averysumner** [comment]
Can you send client and server logs?

**Comment by badooga** [comment]
Unfortunately I don't have any logs because of how long ago this was, but I can provide a screenshot showing the issue. The first screenshot shows that when looking at a given property, a character's other properties will also show up on the list. And the second screenshot shows that this list is the same no matter which property you're looking at.

![image](https://user-images.githubusercontent.com/33851191/151817819-953b76ac-b9bb-4bbf-a467-eca17b7d97b1.png)

![image](https://user-images.githubusercontent.com/33851191/151817906-642cf908-fddf-44eb-b347-8de4c2f8f67c.png)

**Comment by averysumner** [comment]
> Unfortunately I don't have any logs because of how long ago this was, but I can provide a screenshot showing the issue. The first screenshot shows that when looking at a given property, a character's other properties will also show up on the list. And the second screenshot shows that this list is the same no matter which property you're looking at.
> 
> ![image](https://user-images.githubusercontent.com/33851191/151817819-953b76ac-b9bb-4bbf-a467-eca17b7d97b1.png)
> 
> ![image](https://user-images.githubusercontent.com/33851191/151817906-642cf908-fddf-44eb-b347-8de4c2f8f67c.png)

Well can you try again on the latest commit?

**Comment by badooga** [comment]
> Well can you try again on the latest commit?

Yes, once I have some time to test it again I'll send the results and logs here.
